### PR TITLE
unpins pydata sphinx theme

### PR DESCRIPTION
**Pull request by tl3119**
## Issue number

Closes #X

## Checklist before requesting a review

- [x] Performed a self-review of my code
- [x] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)
- [x] Added [tests](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#testing) (when necessary).
- [x] Added [docstring](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#documenting-changes-and-new-features) documentation and update the [changelog](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--573.org.readthedocs.build/en/573/

### CODE REVIEW

File: src/sql/magic.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 import sql.connection
 import sql.parse
 import sql.run
+from sql.parse import _option_strings_from_parser
```
Comment (edublancas) [inline src/sql/magic.py:25]: you're mixing stuff from different PRs. please open a PR that only modifies the requested file

File: doc/environment.yml [hunk 2]
```diff
@@ -25,8 +25,6 @@ dependencies:
     - pytest
     # for %%sql --interact
     - ipywidgets
-    # 0.13.2 is broken currently
     # https://github.com/pydata/pydata-sphinx-theme/issues/1274
```
Comment (edublancas) [inline doc/environment.yml:28]: also remove this line

**Review by edublancas — CHANGES_REQUESTED** [review]

File: src/sql/magic.py (hunk 1 above)
Comment (tl3119) [reply src/sql/magic.py]: Fix it now

File: doc/environment.yml (hunk 2 above)
Comment (tl3119) [reply doc/environment.yml]: Got it

**Review by edublancas — APPROVED** [review]

**Pull request merged**